resources/ext.wikibase.facetedsearch.js

```diff
@@ -57,7 +100,41 @@ function buildQueryString( oldQuery, facet ) {
 		// TODO: Support OR values
 		queries.push( `haswbfacet:${ propertyId }=${ checkbox.value }` );
 	} );
+	return queries;
+}
+
+/**
+ * Constructs an array of range facet queries based on the provided minimum
+ * and maximum input values.
+ *
+ * @param {HTMLInputElement} minInput - The input element for the minimum range value.
+ * @param {HTMLInputElement} maxInput - The input element for the maximum range value.
```

Comment:
Looks like the code only needs the value, not the HTML elements. That will also make it easier to test.

I like that you documented all the types. Dunno about the descriptions tho. There is no requirement to (not) add them. Best to omit if they do not add value (by saying anything not obvious)